boards/v2/intel/intel_ish/intel_ish_5_6_0_defconfig

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023-2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# uart & console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+CONFIG_SERIAL=y
```

Comment:
Because there are all the same, with a revision, you can have a single defconfig instead of 3

Reply:
before those, definition of SoC is different.
With more features added, they may have more differences later.